types/skeleton-rendezvous/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for skeleton-rendezvous 1.1
+// Project: https://github.com/SocketCluster/skeleton-rendezvous#readme
+// Definitions by: aikoven <https://github.com/aikoven>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace SkeletonRendezvousHasher {
+    interface Options {
+        fanout?: number;
+        hashAlgorithm?: string;
+        targetClusterSize?: number;
+        minClusterSize?: number;
+        sites?: string[];
+    }
+}
+
+// eslint-disable-next-line no-redeclare
+declare class SkeletonRendezvousHasher {
+    constructor(options?: SkeletonRendezvousHasher.Options);
+    getSites(): string[];
+    setSites(sites: string[]): void;
+    addSites(sites: string[]): void;
+    removeSites(sites: string[]): void;
+    findSite(key: string): string;
```

Comment:
```suggestion
    findSite(key: string, salt?: string | number): string;
```

(the salt is technically anything that has a .toString() method)

https://github.com/SocketCluster/skeleton-rendezvous/blob/master/index.js#L122

Reply:
This one is not too obvious to me.

I didn't include it because they don't use the second argument anywhere in the tests.

And in the implementation there is incrementing the `salt` by one:
https://github.com/SocketCluster/skeleton-rendezvous/blob/master/index.js#L150
But then there is also concatenating the `salt` with strings:
https://github.com/SocketCluster/skeleton-rendezvous/blob/master/index.js#L153

Reply:
My reading of the JS code is `findSite(key:string, salt?: number)`, but I'm not familiar with the library or how it's used in practice.

Reply:
I've been using this library for quite some time, but I too have no idea of what the `salt` arg stands for. My hypothesis is that it is some internal thing maybe, since this method calls itself recursively with this arg.

So I think we can omit it for now, and maybe add it later if there's demand.

Reply:
Fine by me.